src/components/CvViewer/CvViewer.tsx

```diff
@@ -0,0 +1,17 @@
+import { ResumeViewerType } from './CvViewer.types';
+import { Basic } from './Templates/Basic/Basic';
+import { Modern } from './Templates/Modern/Modern';
+
+type CvViewerProps = {
+  cv: ResumeViewerType;
+};
+export const CvViewer = ({ cv }: CvViewerProps) => {
+  switch (cv.template) {
+    case 'basic reverse':
```

Comment:
Maybe here instead of plain strings use an _enum Template_

Reply:
will do in next pr